### PR TITLE
Space-Travelers-Hub Group Project 

### PR DESCRIPTION
**Pull request by BenMKT**
- Complete Space-Travelers-Hub Group App Project.

### CODE REVIEW

File: src/components/Routes/Rockets.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  return (
+    <div>
+      <br />
+      <ul>
+        {rockets.map((rocket) => (
+          <RocketItem key={rocket.id} rocket={rocket} />
+        ))}
+      </ul>
+    </div>
```
Comment (Olamarx) [inline src/components/Routes/Rockets.jsx:28]: - [ ] Good job creating the rocket components, I noticed an unusual result when it comes to this component. Whenever the Rockets page is refreshed, all the state is reset, kindly ensure that once the user refreshes the Rockets page, the state does not have to be reset. This applies to the mission page too.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Good use of React-router-dom ✔️
- Good use Redux ✔️

## Required Changes ♻️
_Check the comments under the review._
- [x] I could not find tests written, kindly remember that tests for the functions and components are required too 
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Routes/Rockets.jsx (hunk 1 above)
Comment (tomasesquivelgc) [reply src/components/Routes/Rockets.jsx:28]: Hello @Olamarx , regarding this. In your previous comment says we shouldn't preserve user data, so when the page is reloaded the state will inevitably reset.
Furthermore as seen in this picture:
image
Even after several rotations and clicks the data is only fetched once.
It becomes clear on the [LIVE VERSION](https://guileless-halva-645039.netlify.app) that the page suffers no stutters or reloading.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @BenMKT and @tomasesquivelgc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**